build.gradle

```diff
@@ -428,8 +428,11 @@ subprojects {
         sign publishing.publications.maven
     }
 
-    publishMavenPublicationToMavenRepository.onlyIf {
-        !name.contains("grpc-gae-interop-testing") && !name.contains("grpc-xds")
+    // skip publishing incubatingArtifacts when -PskipIncubatingArtifacts=true
+    if (rootProject.properties.get('skipIncubatingArtifacts', false).toBoolean()) {
+        publishMavenPublicationToMavenLocal.onlyIf {
```

Comment:
I'm surprised that only MavenLocal is included.

I'm also surprised we don't do this by default.

Reply:
discussed offline, for current release skip generating artifacts for xds and gae. Filed an issue(#5394) to move this logic in to each subproject.